src/custom-types.json

```diff
@@ -813,6 +813,13 @@
       "https://en.wikipedia.org/wiki/Processing_(programming_language)"
     ]
   },
+  "text/x-rst": {
+    "compressible": true,
+    "extensions": ["rst"],
+    "sources": [
+      "https://docutils.sourceforge.io/rst.html"
```

Comment:
So if this doesn't mention the mime type, it would not be a source for the mime type definition. If you want to submit to this module (as opposed to the IANA), we need a first-party source for the mime type.